server/src/immich/config/profile-image-upload.config.ts

```diff
@@ -25,7 +25,7 @@ function fileFilter(req: AuthRequest, file: any, cb: any) {
     return cb(new UnauthorizedException());
   }
 
-  if (file.mimetype.match(/\/(jpg|jpeg|png|heic|heif|dng|webp)$/)) {
+  if (file.mimetype.match(/\/(jpg|jpeg|png|heic|heif|dng|webp|avif)$/)) {
```

Comment:
So this check is for profile images. I don't know what restrictions there are or why. Would be good to check.

Reply:
Yea, I wasn't sure either, but I figured "if HEIF is there, and DNG is there, AVIF can be there too" :-)

I saw @alextran1502 mention somewhere that AVIF couldn't be used across the board for things like thumbnail images due to inconsistent AVIF support across the various platforms. Does anyone know more details about this so we have a better idea of what limitations we have to cope with?